src/components/authorization/policies/by-role/project-manager.policy.ts

```diff
@@ -45,6 +53,26 @@ import { inherit, member, Policy, Role, sensMediumOrLower } from '../util';
     r.PeriodicReport.read.when(member).edit,
     r.Producible.edit.create,
     r.Product.read.when(member).edit.create.delete,
+    r.ProgressReport.when(member).edit,
+    [r.ProgressReportCommunityStory].flatMap((it) => [
+      it.read,
+      it.when(member).create,
+      it.specifically((p) => [
+        p.responses.whenAll(sensOnlyLow, variant('fpm', 'published')).read,
+        p.responses.whenAll(member, variant('translated')).read,
+        p.responses.whenAll(member, variant('fpm')).edit,
+      ]),
+    ]),
+    [r.ProgressReportHighlight].flatMap((it) => [
+      it.read,
+      it.when(member).create,
+      it.specifically((p) => [
+        p.responses.whenAll(sensMediumOrLower, variant('fpm', 'published'))
+          .read,
```

Comment:
Are FPM's supposed to be able to read notes of other FPM's on project's that are not theirs?
Also are highlights intentionally allowing medium but not community stories?

Reply:
I'm double-checking that I read the spreadsheet correctly (and/or that it is correct itself) for the first question.  For the second, that was a cut/paste error.  They should both be low.